### PR TITLE
Refs Bug#391632 List Group rooms of chatbot

### PR DESCRIPTION
**Pull request by tamtt-1031**
## Related Tickets

- [#391632](https://dev.sun-asterisk.com/issues/391632)

## What's this PR do ?

- [x] List Group rooms of chatbot

## Library
*(List package, library third party add new include version)*

- N/A
## Performance

- [ ] Resolved n + 1 query
- [ ] Run explain query already
- [ ] Time run rake task : 1000 ms

## Checklist

- [x] It was tested in local success?
- [x] Fill link PR into ticket and the opposite
- [x] Note purpose/reason, solution, scope of influence into ticket

## ENV note

```
```

## Notes
*(Other notes)*
Screenshot from 2019-12-06 13-54-32

### CODE REVIEW

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>phpunit</td><td>success</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>failure</td><td></td></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr></table>

File: app/Http/Controllers/RoomController.php [hunk 1]
```diff
@@ -19,6 +19,13 @@ public function index(Request $request)
         $bot = Bot::findOrFail($request->bot_id);
         $chatwork = Chatwork::withAPIToken($bot->bot_key);
         $rooms = $chatwork->rooms()->list();
-        return $rooms;
+        $group_boxs = [];
```
Comment (framgiaciv3) [inline app/Http/Controllers/RoomController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Controllers/RoomController.php</br>**Line**: 22</br>Variable "group_boxs" is not in valid camel caps format

File: app/Http/Controllers/RoomController.php [hunk 2]
```diff
@@ -19,6 +19,13 @@ public function index(Request $request)
...
+        foreach($rooms as $room) {
```
Comment (framgiaciv3) [inline app/Http/Controllers/RoomController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterKeyword</br>**Severity**: error</br>**File**: app/Http/Controllers/RoomController.php</br>**Line**: 23</br>Expected 1 space(s) after FOREACH keyword; 0 found

File: app/Http/Controllers/RoomController.php [hunk 3]
```diff
@@ -19,6 +19,13 @@ public function index(Request $request)
...
+            if ($room['type'] == "group") {
```
Comment (framgiaciv3) [inline app/Http/Controllers/RoomController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Strings.DoubleQuoteUsage.NotRequired</br>**Severity**: error</br>**File**: app/Http/Controllers/RoomController.php</br>**Line**: 24</br>String "group" does not require double quotes; use single quotes instead

File: app/Http/Controllers/RoomController.php [hunk 4]
```diff
@@ -19,6 +19,13 @@ public function index(Request $request)
...
+                array_push($group_boxs, $room);
```
Comment (framgiaciv3) [inline app/Http/Controllers/RoomController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Controllers/RoomController.php</br>**Line**: 25</br>Variable "group_boxs" is not in valid camel caps format

File: app/Http/Controllers/RoomController.php [hunk 5]
```diff
@@ -19,6 +19,13 @@ public function index(Request $request)
...
+            }
+        }
+
+        return $group_boxs;
```
Comment (framgiaciv3) [inline app/Http/Controllers/RoomController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Controllers/RoomController.php</br>**Line**: 29</br>Variable "group_boxs" is not in valid camel caps format

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>success</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr></table>

**Review by ghost — APPROVED** [review]

**Review by nguyenthinhhn — APPROVED** [review]

**Comment by phucqd-1305** [comment]
OK

**Pull request merged**